src/actions/actionTypes.js

```diff
@@ -31,11 +31,13 @@ const actionTypes = {
   GET_MORE_ARTICLES: "GET_MORE_ARTICLES",
   STORE_PARAM: "STORE_PARAM",
   GET_MY_BOOKMARKS: "MY_BOOKMARKS",
-  COMMENTSUCCESS: 'COMMENTSUCCESS',
-  COMMENTERROR: 'COMMENTERROR',
-  REPLYSUCCESS: 'REPLYSUCCESS',
-  GETCOMMENT: 'GETCOMMENT',
-  EDITCOMMENT: 'EDITCOMMENT',
-  COMMENTDELETE: 'COMMENTDELETE',
+  COMMENTSUCCESS: "COMMENTSUCCESS",
+  COMMENTERROR: "COMMENTERROR",
+  REPLYSUCCESS: "REPLYSUCCESS",
+  GETCOMMENT: "GETCOMMENT",
+  EDITCOMMENT: "EDITCOMMENT",
+  COMMENTDELETE: "COMMENTDELETE",
+  READ_TIME: "READ_TIME",
+  MY_ARTICLES: "MY_ARTICLES"
```

Comment:
`MY_ARTICLES` is not a proper word for an action. It should include a verb implying something that is done @KicaRonaldOkello